ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -41,6 +41,12 @@ protocol DisplayViewEditingDelegate: class {
 class DisplayViewController: UIViewController {
 
 	private let iconImageSize: CGSize = CGSize(width: 200.0, height: 200.0)
+	private let iconImageViewCenterYConstraintConstant: CGFloat = -60.0
```

Comment:
@pablocarmu I am not convinced by creating constants just for the sake of creating constants. For example, i would ask a question in an expression like this `cancelButton!.topAnchor.constraint(equalTo: progressView!.bottomAnchor, constant: cancelButtonTopConstraintConstant`: what is `cancelButtonTopConstraintConstant`? Is it spacing between UI elements? Is it some margin etc.? I mean I would expect something more generic.